packages/terra-functional-testing/tests/jest/logger/logger.test.js

```diff
@@ -0,0 +1,24 @@
+const Logger = require('../../../lib/logger/logger');
+
+describe('Logger', () => {
+  describe('format', () => {
+    it('should format test with the initialized prefix', async () => {
```

Comment:
Can we mock the console.log for this whole test to prevent log messages during the test run?

Also do these test need an async callback?